### PR TITLE
Imp 193 pylint fixes pylint strikes back

### PR DESCRIPTION
**Pull request by wxtim**
Reference issue if exists

Testing:
 - [ ] Ran tests and they passed OK
 - [ ] Added new tests for the new feature(s)

CLA
 - [ ] If a new developer, signed up to CLA

### CODE REVIEW

**Comment by bayliffe** [comment]
There is still a missing doc-string in:
Module improver.tests.ensemble_calibration.ensemble_calibration.test_ApplyCoefficientsForEnsembleCalibration for the function datetime_from_timestamp

**Comment by bayliffe** [comment]
There is a comment that doesn't belong to anything in test_ContinuousRankedProbabilityScoreMinimisers.py on line 485.

**Comment by bayliffe** [comment]
absolute_differences_between_adjacent_grid_squares takes a threshold argument that does not appear to be used; needs checking.

**Comment by bayliffe** [comment]
Unused DimCoord import in spatial.py

File: lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py [hunk 1]
```diff
@@ -328,11 +312,9 @@ def test_basic_normal_members_predictor(self):
                      5.55444682e-01, 5.04367388e-01, 6.68575194e-01])
 
     def test_normal_mean_predictor_keyerror(self):
-        """
-        Test that the minimisation has resulted in a KeyError, if the
+        """Test that the minimisation has resulted in a KeyError, if the
```
Comment (fionaRust) [inline lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py]: FYI if you look in the style guide https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-test-structure it says that unit tests should have a titile like like this
```
 """Some kind of title"""
```
not like 
```
"""
Some kind of title
"""
```

This is so it shows up as a title in the unit test output.

**Pull request closed without merging**